ml/src/main/scala/py_sparkling/ml/algos/H2OAutoML.scala

```diff
@@ -16,27 +16,11 @@
 */
 package py_sparkling.ml.algos
 
-import ai.h2o.automl.AutoML
-import org.apache.spark.ml.h2o.algos.H2OAutoMLReader
-import org.apache.spark.ml.util.{MLReadable, MLReader}
-import py_sparkling.ml.models.H2OMOJOModel
-import water.support.ModelSerializationSupport
+import org.apache.spark.ml.util.DefaultParamsReadable
 
 /**
   * H2O AutoML Wrapper for PySparkling
   */
-class H2OAutoML(override val uid: String) extends org.apache.spark.ml.h2o.algos.H2OAutoML(uid) {
+class H2OAutoML(override val uid: String) extends org.apache.spark.ml.h2o.algos.H2OAutoML(uid)
 
-  override def trainModel(aml: AutoML): H2OMOJOModel = {
```

Comment:
The implementation is the same as in the super class. We can remove it. The same holds for all others PySparkling wrappers

Reply:
That sounds great!